Makefile

```diff
@@ -61,12 +52,49 @@ BUILDFILE_AD_SERVER:=$(BUILDFILE_PREFIX).ad-server
 BUILDFILE_NIGHTLY_AD_SERVER:=$(BUILDFILE_PREFIX).nightly-ad-server
 BUILDFILE_CLIENT:=$(BUILDFILE_PREFIX).client
 BUILDFILE_TOOLBOX:=$(BUILDFILE_PREFIX).toolbox
+OS_PREFIX=$(addsuffix -,$(OS_NAME))
+TAG=$(OS_PREFIX)latest
+NIGHTLY_TAG=$(OS_PREFIX)nightly 
+
+
+SERVER_NAME=samba-server:$(TAG)
+NIGHTLY_SERVER_NAME=samba-ad-server:$(NIGHTLY_TAG)
+AD_SERVER_NAME= samba-ad-server:$(TAG)
+NIGHTLY_AD_SERVER_NAME=samba-ad-server:$(NIGHTLY_TAG)
+CLIENT_NAME=samba-client:$(TAG)
+NIGHTLY_CLIENT_NAME=samba-client:$(NIGHTLY_TAG)
+TOOLBOX_NAME=samba-toolbox:$(TAG)
+NIGHTLY_TOOLBOX_NAME=samba-toolbox:$(NIGHTLY_TAG)
+
+REPO_BASE=quay.io/samba.org/
+SERVER_REPO_NAME=$(REPO_BASE)$(SERVER_NAME)
+NIGHTLY_SERVER_REPO_NAME=$(REPO_BASE)$(NIGHTLY_SERVER_NAME)
+AD_SERVER_REPO_NAME=$(REPO_BASE)$(AD_SERVER_NAME)
+NIGHTLY_AD_SERVER_REPO_NAME=$(REPO_BASE)$(NIGHTLY_AD_SERVER_NAME)
+CLIENT_REPO_NAME=$(REPO_BASE)$(CLIENT_NAME)
+NIGHTLY_CLIENT_REPO_NAME=$(REPO_BASE)$(NIGHTLY_CLIENT_NAME)
+TOOLBOX_REPO_NAME=$(REPO_BASE)$(TOOLBOX_NAME)
+NIGHTLY_TOOLBOX_REPO_NAME=$(REPO_BASE)$(NIGHTLY_TOOLBOX_NAME)
+
+BUILDFILE_PREFIX=.build
+BUILDFILE_SERVER=$(BUILDFILE_PREFIX).$(OS_PREFIX)server
+BUILDFILE_NIGHTLY_SERVER=$(BUILDFILE_PREFIX).$(OS_PREFIX)nightly-server
+BUILDFILE_AD_SERVER=$(BUILDFILE_PREFIX).$(OS_PREFIX)ad-server
+BUILDFILE_NIGHTLY_AD_SERVER=$(BUILDFILE_PREFIX).$(OS_PREFIX)nightly-ad-server
+BUILDFILE_CLIENT=$(BUILDFILE_PREFIX).$(OS_PREFIX)client
+BUILDFILE_NIGHTLY_CLIENT=$(BUILDFILE_PREFIX).$(OS_PREFIX)nightly-client
+BUILDFILE_TOOLBOX=$(BUILDFILE_PREFIX).$(OS_PREFIX)toolbox
+BUILDFILE_NIGHTLY_TOOLBOX=$(BUILDFILE_PREFIX).$(OS_PREFIX)nightly-toolbox
+>>>>>>> 33d4da0 (Makefile: generalize build mechanisms - prepare for adding more)
 
 build: build-server build-nightly-server build-ad-server build-client \
 	build-toolbox
 .PHONY: build
 
 
+
+
+
```

Comment:
Unnecessary whitespace change

Reply:
Probably doesn't matter as the next patch cleans this up anyway (when debug-vars gets added).